package.json

```diff
@@ -4,6 +4,8 @@
   "description": "",
   "main": "content_script.js",
   "scripts": {
+    "reload": "open http://reload.extensions/",
+    "dev": "chokidar \"src/*.js\" -c \"npm run build;npm run reload\"\n\n",
```

Comment:
Can we change this to `start`?

Reply:
Sure thing. That's what I had at first but I thought start had some kind of server feel to it.